microsoft/botframework-solutions
# Enterprise Bot Tests

## Description
Close 
### Maintenance
Added unit tests to the Enterprise Template:
•	Intro Card
•	Greeting
•	Help
•	Escalate
•	Onboarding
•	Onboarding_Cancel
•	Intro card (localised)

## Testing Steps

1. Copy EnterpriseBotTemplate folder contents to "%USERPROFILE%\Documents\Visual Studio 2017\Templates\ProjectTemplates\Visual C#"
1. In Visual Studio, select File > New > Project
1. Click the "Enterprise Bot Template" in the Bot category
1. Rebuild the project (to copy content files to Test project output)
1. Verify all tests pass

## Checklist
